sources/api/thar-be-settings/src/service.rs

```diff
@@ -135,8 +155,21 @@ impl ServiceRestart for model::Service {
             trace!("Args: {:?}", &command_strings);
 
             // Go execute the restart command
-            let result = process::Command::new(command)
-                .args(command_strings)
+            let mut process_command = process::Command::new(command);
+            process_command.args(command_strings);
+            match changed_settings {
+                Some(value) => {
+                    let changed_settings_string = value.iter().join(" ");
+                    info!(
+                        "restarting service  with changed settings {:?}",
```

Comment:
nit: I think we should move this log message one level up to `restart_services` where we have the service name and the list of settings. Let's remove it from here.

Reply:
Also, do you think it would be good idea to set empty string value to  key `changed_settings` key for `.env`  if there are no `changed_settings` so that we can get rid of `Option` for `changed_settings` in this function signature. 
Currently I am not setting `.env` for the services if `changed_settings` is empty

Reply:
I think that makes sense.